CryptoLightNews/Networking/Service.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 class Service {
     
     typealias News = Result<SearchResults, CryptoNewsErrors>
+    typealias Coins = Result<SearchResultsCoins, CryptoNewsErrors>
```

Comment:
I suggest you name it ResultCoins since you can have an error type here.
```suggestion
    typealias ResultCoins = Result<Coins, CryptoNewsErrors>
```